src/test/java/seedu/commando/logic/parser/SequentialParserTest.java

```diff
@@ -214,4 +219,22 @@ public void extractTrailingDateRange_onlyTrailing() throws IllegalValueException
             "walk by the beach from today to tomorrow", sequentialParser.extractText().orElse("")
         );
     }
+    @Test
+    public void extractIndicesList_valid() throws IllegalValueException {
+    	sequentialParser.setInput("2to7");
+    	List<Integer> indices = sequentialParser.extractIndicesList();
+    	assertEquals("[2, 3, 4, 5, 6, 7]",indices.toString());
+    	sequentialParser.setInput("-2-7");
+    	indices = sequentialParser.extractIndicesList();
+    	assertEquals("[-2, -1, 0, 1, 2, 3, 4, 5, 6, 7]",indices.toString());
+    	sequentialParser.setInput("-2  -  7");
+    	indices = sequentialParser.extractIndicesList();
+    	assertEquals("[-2, -1, 0, 1, 2, 3, 4, 5, 6, 7]",indices.toString());
+    	sequentialParser.setInput("1 2 3 4 5");
+    	indices = sequentialParser.extractIndicesList();
+    	assertEquals("[1, 2, 3, 4, 5]",indices.toString());
+    	sequentialParser.setInput("2to2");
```

Comment:
Think you should also test:
- more common use case of `"2 to 7"`
- catching an exception from `"2 to -2"`
- just a single `"2"`